astroquery/utils/tap/model/tests/test_job.py

```diff
@@ -26,13 +26,13 @@ def data_path(filename):
     data_dir = os.path.join(os.path.dirname(__file__), 'data')
     return os.path.join(data_dir, filename)
 
-class TestJob(unittest.TestCase):
 
+class TestJob(unittest.TestCase):
 
     def test_job_basic(self):
         job = Job(async=False)
         res = job.is_sync()
-        assert res == True, \
```

Comment:
also I think this is a bit superfluous to have one liners like this, but that's another story